language_detector/main.py

```diff
@@ -5,5 +5,16 @@
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    text_list = text.split()
+    #print(text_list)
```

Comment:
It's not a good practice to submit old commented code. Please remove unnecessary lines instead.